### ISSUE TITLE
TypeError: axiosETAGCache is not a function with example code

### ISSUE THREAD

**Issue by Cactusbone** [raulanatol/axios-etag-cache#36]
Tested on version 1.1.1

See https://runkit.com/cactusbone/axios-etag-cache/1.0.0

Example code does not run in nodejs (tested on v16 and v14).

We get the error: `TypeError: axiosETAGCache is not a function`

Looking at returned value, we get an object and not a function
```
{
  BaseCache: [Getter],
  Cache: [Getter],
  DefaultCache: [Getter],
  resetCache: [Getter]
}
```

looking at the built `main.js`, there is no way to require it:
```
(module.exports = {BaseCache, Cache, DefaultCache, resetCache})
```

Looking at version 1.0.5 it was possible to use `const axiosETAGCache = require('axios-etag-cache@1.0.5').default;` (still not the example), but it's not exported anymore.

**Comment by raulanatol** [comment]
@Cactusbone thanks for your notice.

I've fixed this issue with #38 and I will close a release ASAP.

**Comment by PoojaBenbansi** [comment]
I am using index.test.ts file but 5 test cases are failing where we are testing for api calls
